st2reactor/tests/integration/test_sensor_watcher.py

```diff
@@ -31,6 +31,15 @@ class SensorWatcherTestCase(IntegrationTestCase):
     def setUpClass(cls):
         super(SensorWatcherTestCase, cls).setUpClass()
 
+    def setUp(self):
+        super().setUp()
+        # pre-condition: Make sure there is no test pollution
+        sw_queues = self._get_sensor_watcher_amqp_queues(
+            queue_name="st2.sensor.watch.covfefe"
+        )
+        # TODO: Maybe just delete any leftover queues from previous failed test runs.
+        self.assertTrue(len(sw_queues) == 0)
```

Comment:
This change is unrelated to this PR. This test failed for several runs, and now it is magically passing.

Apparently we still have some cross test pollution with data in RabbitMQ. So, I'll leave this here and next time we see the bug, hopefully we can figure out how to prevent that cross test pollution that can make tests randomly fail.

I'm wondering about having some pytest fixture handle cleaning up all exchanges/queues for the execution slot before running any tests. If we do that, then we'd need to make sure not to break the tests that use the full st2 cluster (started with `tools/launchdev.sh`).